src/frontends/lean/elaborator.cpp

```diff
@@ -2349,7 +2349,8 @@ static void mvar_dep_sort(type_context & ctx, buffer<expr> & mvars) {
    Before processing the right-hand-side, we must convert back these metavariables into
    pattern variables (i.e., local constants).
    This visitor collects metavariables that must be converted into pattern variables, and
-   validates the equation left-hand-side. */
+   validates the equation left-hand-side. It returns the input expression expanded to
+   expose its primitive patterns. */
```

Comment:
@leodemoura This should be correct, yes?

Reply:
Correct.